include/eve/conditional.hpp

```diff
@@ -533,4 +544,17 @@ namespace eve
     if constexpr (!C::has_alternative) return c;
     else                               return c.map_alternative(map);
   }
+
+  template <eve::relative_conditional_expr C, typename T>
+  constexpr auto reverse_conditional(C c, eve::as<T> tgt)
+  {
+         if constexpr ( C::is_complete                  ) return c;
```

Comment:
I was about to comment on this to be too closed for extension but we agree we have all the ignore* we need anyway right ?